src/sentry/buffer/base.py

```diff
@@ -83,7 +106,7 @@ def process(self, model, columns, filters, extra=None, signal_only=None):
                     group.update(**update_kwargs)
                 created = False
             else:
-                _, created = model.objects.create_or_update(values=update_kwargs, **filters)
+                _, created = model.objects.create_or_update(values=update_kwargs, **filters)  # type: ignore[attr-defined]
```

Comment:
ah, this hints that we're using sentry-models-subclasses -- so you can probably use `sentry.db.models.Model` instead of `django.models.Model` in the type here and remove the ignore

Reply:
Nice. Thanks!